kaldb/src/main/java/com/slack/kaldb/chunkManager/RollOverChunkTask.java

```diff
@@ -40,25 +40,30 @@ public RollOverChunkTask(
     rolloversFailedCounter = meterRegistry.counter(ROLLOVERS_FAILED);
   }
 
+  // TODO: Is the return value needed?.
```

Comment:
Not sure of the purpose of this todo? The only way we communicate a runtime exception with the refactored code is via this boolean, which is checked in the IndexingChunkManager `onSuccess`